composer.json

```diff
@@ -20,7 +20,7 @@
         "composer/installers": "^1.2",
         "drupal-composer/drupal-scaffold": "^2.2",
         "cweagans/composer-patches": "~1.0",
-        "drupal/core": "~8.0",
+        "drupal/core": "8.2.7",
```

Comment:
Should we do 8.2.* so we get security patches for the 8.2 series if there happen to be any?